src/Mod/CAM/Path/Op/Gui/CircularHoleBase.py

```diff
@@ -162,17 +162,26 @@ def registerSignalHandlers(self, obj):
         self.form.addBase.clicked.connect(self.addBase)
         self.form.deleteBase.clicked.connect(self.deleteBase)
         self.form.resetBase.clicked.connect(self.resetBase)
+        self.form.clearBase.clicked.connect(self.clearBase)
         self.form.baseList.itemChanged.connect(self.checkedChanged)
 
     def resetBase(self):
-        """resetBase() ... push button callback"""
+        """resetBase() ... Reset push button callback"""
         self.obj.Base = []
         self.obj.Disabled = []
         self.obj.Proxy.findAllHoles(self.obj)
 
         self.obj.Proxy.execute(self.obj)
         FreeCAD.ActiveDocument.recompute()
 
+    def clearBase(self):
+        print("entering clearBase")
```

Comment:
Did you forget to remove this debug print?

Reply:
Thanks. Well spotted. 
I've added a new commit to my fork:
https://github.com/FreeCAD/FreeCAD/commit/65fba5fc628830d11050b44e82db9307314fc641

How can I push this without triggering a new PR? 
Is GH even capable of doing that?

Reply:
GH updates the PR automatically when you push changes to the same branch.
So your change is already in this PR :)

Reply:
cool . Thanks for the help.